metacatalog/models/entry.py

```diff
@@ -683,7 +684,7 @@ def add_details(self, details=None, commit=False, **kwargs):
                 session.rollback()
                 raise e
 
-    def export(self, path=None, fmt='JSON', **kwargs):
+    def export(self, path: str = None, fmt: str = 'JSON', **kwargs):
```

Comment:
`-> None` ?

Reply:
Nope. It does return something, but I have no idea how to pipe the correct types through to this function.
I guess we keep it like it is, or do that later